ci/kind/kind-setup.sh

```diff
@@ -244,25 +245,68 @@ function configure_extra_networks {
   done
 }
 
+function get_bridge_id {
+  local bridge_id=$(docker network inspect kind -f {{.ID}})
+  local return_value=$?
+  if [ $return_value -eq 0 ]; then
+        echo "$bridge_id"
+    else
+        echo "Unable to get kind bridge id, Docker command failed with exit code $return_value"
+        exit 1
+    fi
+}
+
+# add_kind_ipam_routes  add routes for non-ipam test-pods.
+function add_kind_ipam_routes {
+     kubectl get nodes -o wide --no-headers=true |  awk '{print $1}' | while read nodes; do
+      local pod_cidr=$(kubectl get nodes "${nodes}" -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+      local node_ip=$(kubectl get nodes "${nodes}" -o wide | awk 'FNR == 2 {print$6}')
+      sudo ip route add "$pod_cidr" via "$node_ip"
+     done
+}
+
+# delete_kind_ipam_routes delete the routes of non-ipam test-pods from the Kind testbed.
+function delete_kind_ipam_routes { 
+    cluster=$(kind get clusters)
+    if [ ! -z "$cluster" ]; then
+        cidrs=$(kubectl get nodes -o jsonpath='{.items[*].spec.podCIDR}')
+          echo -e "$cidrs" | tr ' ' '\n'| while read cidr; do
+                del_routes=$(ip route | awk -v cidr="$cidr" '$0 ~ cidr {print$1}')
+                if [[ -z $del_routes ]]; then
+                    echo "Test pods routes are not available on testbed"
+                    exit 0
```

Comment:
I don't think this is right. The function will execute in all cluster and apparently not all of them has the route. This will terminate other testbed's cleanup.

Why doesn't it just remove the route when there is?